android/app/src/main/res/layout/toolbar_search_controls.xml

```diff
@@ -11,13 +11,14 @@
   android:gravity="center_vertical">
 
   <ImageView
-    android:id="@+id/back"
-    android:layout_width="?attr/actionBarSize"
-    android:layout_height="?attr/actionBarSize"
-    android:background="?attr/selectableItemBackgroundBorderless"
-    app:srcCompat="?homeAsUpIndicator"
-    android:scaleType="center"
-    tools:src="@drawable/ic_down" />
+      android:id="@+id/back"
```

Comment:
Why indentation is reformatted? We use 2 spaces everywhere.

Reply:
Fixed ^^